src/Kurrent.Client/Core/Serialization/SchemaRegistry.cs

```diff
@@ -0,0 +1,93 @@
+using EventStore.Client;
+using EventStore.Client.Serialization;
+using Kurrent.Client.Tests.Streams.Serialization;
+
+namespace Kurrent.Client.Core.Serialization;
```

Comment:
Can we actually have one namespace?

Reply:
Fixed in https://github.com/EventStore/EventStore-Client-Dotnet/pull/329/commits/e8385b38d24f5eb138b8393a4acbcdc0c6416a9b